src/index.html

```diff
@@ -4,65 +4,375 @@
 <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Stake'n'Vote</title>
+    <title>Kusamarian Stake'n'Vote</title>
     <style>
         @import "/src/styles.css";
     </style>
 </head>
-<body>
+<body onload="preLoad()">
     <nav class="navbar">
         <div class="navbar-brand">
-            <a class="navbar-item is-size-3 has-text-weight-semibold" href="#">Stake'n'Vote</a>
+            <a class="navbar-item is-size-3 has-text-weight-semibold" href="#"><img class="logo" src="logos/kulogo.png" alt=""></a>
         </div>
         <div class="navbar-end">
             <my-address-chooser id="address-chooser"></my-address-chooser>
         </div>
     </nav>
+
+    <!-- -------------- justadev ------------------------- -->
+
     <section>
         <div class="container">
-            <div class="columns is-vcentered">
-                <div class="column">
-                    <div class="field">
-                        <input id="stakingSwitch" type="checkbox" name="switchLarge" class="switch is-large is-rtl">
-                        <label for="stakingSwitch">I want to stake with you!</label>
-                      </div>
-                </div>
-                <div class="column">
-                    <my-validator-list id="validator-list"></my-validator-list>
-                </div>
-            </div>
+      
+<div class="buttons-container">
+    <p class="title">SELECT NETWORK</p>
+
+
+    <label class="radio-img">
+      <input type="radio" name="layout" value="L" onClick="im('a1');" />
```

Comment:
I would rather this use `element.addEventListener` like everything else in the app.